guide/resources/design-research.md

```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [2024 American Bitcoin Survey report](https://www.thenakamotoproject.org/report)
+
+_2024 by The Nakamoto Project_
```

Comment:
```suggestion
 Understanding Bitcoin Adoption in the United States: Politics, Demographics, & Sentiment
```

Reply:
It's from the report landing page.

<img width="484" alt="image" src="https://github.com/user-attachments/assets/af086fea-44cc-494a-ad52-aaed7e76e7dd">

Reply:
Yes I see that, it seems to be an example of unintentional formatting and/or poor writing. I would still encourage using the suggested title "Understanding Bitcoin Adoption in the United States: Politics, Demographics, & Sentiment" for following reasons:
- This title appears on the first page of the report (see image below)
- It is more descriptive, precise and accurate characterisation of the research
- It mentions that this was not a general bitcoin survey but centred around adoption
- NP do not seem to have performed this research in 2023 or earlier

<img width="620" alt="Screenshot 2024-08-05 at 11 38 24" src="https://github.com/user-attachments/assets/479a812f-e876-49fa-919d-c6c3ee869aed">